server/service.go

```diff
@@ -116,10 +116,24 @@ func (m *BoostService) handleRoot(w http.ResponseWriter, req *http.Request) {
 	fmt.Fprintf(w, `{}`)
 }
 
+// handleStatus sends calls to the status endpoint of every relay.
+// It returns OK if at least one returned OK, and returns KO otherwise.
 func (m *BoostService) handleStatus(w http.ResponseWriter, req *http.Request) {
-	w.Header().Set("Content-Type", "application/json")
-	w.WriteHeader(http.StatusOK)
-	fmt.Fprintf(w, `{}`)
+	for _, relay := range m.relays {
```

Comment:
You can parallelise the call to check relay status. See https://github.com/flashbots/mev-boost/blob/main/server/service.go#L160-L193 for an example of how it's done.

Reply:
Thanks ! Added in 9d7cab19f8b96cefe73e15e821c516c863ae7457